webapp/graphql/Fragments.js

```diff
@@ -45,6 +45,7 @@ export const postFragment = lang => gql`
     deleted
     slug
     image
+    language
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-14T23:53:08Z" title="Friday, November 15th 2019, 12:53:08 am +01:00">Nov 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -45,6 +45,7 @@ export const postFragment = lang => gql`
     deleted
     slug
     image
+    language
```


How could we have missed that? :thinking: